### PR TITLE
Assessments: add summary text component and story

### PR DESCRIPTION
**Pull request by nkiruka**
This PR is to add a  "SummaryText" component and story.

The summary text component displays the results of how many students submit an assessment.  This component will be displayed above the overview assessment table.

### CODE REVIEW

File: apps/src/templates/sectionAssessments/SummaryText.jsx [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  static propTypes = {
+    numSubmissions: PropTypes.string,
+  };
+
+  render() {
+    const {numSubmissions} = this.props;
+    return (
+      <div>
+        <div>
+          {`multiple choice questions overview ${numSubmissions}`}
```
Comment (caleybrock) [inline apps/src/templates/sectionAssessments/SummaryText.jsx]: nit: uppercase m
(and add to i18n)

**Comment by nkiruka** [comment]
@caleybrock Updated PR

File: apps/src/templates/sectionAssessments/SummaryText.jsx [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    numStudentSubmissions: PropTypes.number.isRequired,
+    totalNumStudents: PropTypes.number.isRequired,
+  };
+
+  render() {
+    const {numStudentSubmissions, totalNumStudents} = this.props;
+    return (
+      <div>
+        <div>
+          {i18n.summaryAssessmentsOverview()} {`${numStudentSubmissions} / ${totalNumStudents}`}
```
Comment (caleybrock) [inline apps/src/templates/sectionAssessments/SummaryText.jsx]: you can do this without double {}:
`{i18n.summaryAssessmentsOverview() + '${numStudentSubmissions} / ${totalNumStudents}'}`

Comment (nkiruka) [reply apps/src/templates/sectionAssessments/SummaryText.jsx]: @caleybrock the string concatenation did not display the text correctly.  I reverted back to the string interpolation I had before.  Thoughts..

Comment (nkiruka) [reply apps/src/templates/sectionAssessments/SummaryText.jsx]: @caleybrock Dave helped out with refactoring code.  All set.

**Pull request merged**